cf/aggregate.py

```diff
@@ -2795,9 +3120,14 @@ def _group_fields(meta, axis):
         axis: `str`
             The name of the axis to group for aggregation.
 
+        info: `bool`
+            True if the log level is ``'INFO'`` (``2``).
+
+            .. versionaddedd:: 3.15.1
+
     :Returns:
 
-        `list` of cf.FieldList
+        `list` of `FieldList`
```

Comment:
It turns out neither is right! b7ba0d35243282b7fbc864073c4b43dde23da2eb

Reply:
Fair enough, the main thing is to get the right one! I appreciate it's an 'internal' method but to avoid confusing ourselves down the line it's always good to keep these accurate...

The fix commit looks good. Marking as resolved. Thanks.